File: JAVA PROGRAMMING/Test29.java
// 29. PRINT MULTIPLICATION TABLE
import java.util.Scanner;
public class Test29 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Input1 Value: ");
        int ip1 = in.nextInt();
        System.out.println("Enter Input2 Value: ");
        int ip2 = in.nextInt();
        for(int i= 0; i <= ip2; i++) {
            System.out.println(ip1+" X "+i+" = "+(ip1 * i));
        }
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test29.ts
// 29. PRINT MULTIPLICATION TABLE
const func29 = (ip: number, target: number): void => {
    for(let i = 0; i <= target; i++) {
        console.log(`${ip} * ${i} = ${ip * i}`);
    }
}
func29(5, 20);
func29(23, 10);